manifests/0000_90_dns-operator_03_prometheusrules.yaml

```diff
@@ -34,3 +34,15 @@ spec:
           severity: warning
         annotations:
           message: "CoreDNS is returning SERVFAIL for {{ $value | humanizePercentage }} of requests."
+      - alert: CoreDNSForwardHealthcheckFailureCount
+        annotations:
+          message: '{{ $value }} A CoreDNS counter for the number of failed health checks for an upstream server on {{ $labels.instance }}'
+        expr: coredns_forward_healthcheck_failure_count_total > 0
```

Comment:
How often are those healthchecks executed? 
By having it fire for >0 this imposes a risk for flapping alerts

Reply:
> How often are those healthchecks executed?

@RiRa12621 the checks are executed every 0.5 sec by default.